mappings/net/minecraft/world/timer/TimerCallback.mapping

```diff
@@ -10,9 +10,9 @@ CLASS net/minecraft/class_234 net/minecraft/world/timer/TimerCallback
 			ARG 1 id
 			ARG 2 callbackClass
 		METHOD method_975 serialize (Lnet/minecraft/class_2487;Lnet/minecraft/class_234;)V
```

Comment:
```suggestion
		METHOD method_975 writeNbt (Lnet/minecraft/class_2487;Lnet/minecraft/class_234;)V
```
Subject to debate, as this is in a specific serializer class than the timer callback itself